pydantic/schema.py

```diff
@@ -807,6 +839,11 @@ def go(type_: Any) -> Type[Any]:
     return ans
 
 
+def normalize_model_name(name: str) -> str:
+    """Normalizes the given model name."""
```

Comment:
```suggestion
    """
    Normalizes the given model name.
    """
```
Sorry, just how I like docstrings.

Reply:
Haha sounds great, fixed!